instrumented/benchmark/src/androidTest/kotlin/com/datadog/android/sdk/benchmark/trace/TraceApiBenchmark.kt

```diff
@@ -56,11 +59,21 @@ class TraceApiBenchmark {
         val fakeEndpoint = mockWebServer.url("/").toString().removeSuffix("/")
 
         val context = InstrumentationRegistry.getInstrumentation().context
-        val config = DatadogConfig
-            .Builder("NO_TOKEN", "benchmark")
+        val config = Configuration
+            .Builder(true, true, true, true)
```

Comment:
maybe we can use named arguments here? hard to understand what each boolean means without looking into source code. also same applies to other places where `Builder` is invoked like that in this PR.